keps/sig-scheduling/785-scheduler-component-config-api/README.md

```diff
@@ -117,6 +117,7 @@ More information on the discussion can be found [here](https://github.com/kubern
 
 The fourth iteration, `kubescheduler.config.k8s.io/v1`, includes the following changes:
   - Mark `v1beta2` as deprecated
+  - Mark `SelectorSpread` plugin as deprecated
```

Comment:
it's not deprecated. It's removed from v1.

Reply:
Or No longer support `SelectorSpread` plugin, will remove the codes in v1.27. We didn't remove the code yet, although end-users can't use it anymore.

Reply:
They can use it in v1beta3. But it's removed from v1. That's the same language we used above for v1beta2.

Reply:
Thanks for explaining. Done:)